site/src/pages/TemplatePage/TemplateSummaryPage/DAUChart.tsx

```diff
@@ -119,16 +119,13 @@ export const DAUChart: FC<DAUChartProps> = ({
               {
                 label: "Daily Active Users",
                 data: data,
-                lineTension: 1 / 4,
+                tension: 1 / 4,
```

Comment:
Unsure if typo or API change but this matches the types and what you see in [Chart.js docs](https://www.chartjs.org/docs/latest/charts/line.html#line-styling)